test/adjust.js

```diff
@@ -0,0 +1,29 @@
+var assert = require('assert');
+
+var R = require('..');
+
+describe('adjust', function() {
+  it('should apply the given function to the value at the given index of the supplied array', function() {
+    assert.deepEqual(R.adjust(R.add(1), 2, [0, 1, 2, 3]), [0, 1, 3, 3]);
+  });
+
+  it('should offset negative indexes from the end of the array', function() {
+    assert.deepEqual(R.adjust(R.add(1), -3, [0, 1, 2, 3]), [0, 2, 2, 3]);
+  });
+
+  it('should return the original array if the supplied index is out of bounds', function() {
+    var list = [0, 1, 2, 3];
+    assert.strictEqual(R.adjust(R.add(1), 4, list), list);
+    assert.strictEqual(R.adjust(R.add(1), -5, list), list);
+  });
+
+  it('should not mutate the original array', function() {
+    var list = [0, 1, 2, 3];
+    assert.deepEqual(R.adjust(R.add(1), 2, list), [0, 1, 3, 3]);
+    assert.deepEqual(list, [0, 1, 2, 3]);
+  });
+
+  it('should curry the arguments', function() {
+    assert.deepEqual(R.adjust(R.add(1))(2)([0, 1, 2, 3]), [0, 1, 3, 3]);
+  });
+});
```

Comment:
We are trying to get away from the `it('should apply...')` style and use the more direct `it('applies ...')`.  While we haven't necessarily gone back and fixed all the old tests, it would be nice not to introduce new ones like this.  (I may have missed this in the Lenses commit.)